### PR TITLE
Rich text component with markdown support.

### PR DESCRIPTION
**Pull request by Cacodaimon**
I teached the richtext component to speak markdown.

I would suggest to bump the version to 0.4.0 due to minor changes.

Feature discussion here: https://github.com/nextcloud/spreed/issues/1027#issuecomment-697408559

### CODE REVIEW

**Comment by juliushaertl** [comment]
Code looks good to me, mind to have a look at the failing documentation build and maybe extend the docs with a page showing the markdown usage?

**Comment by Cacodaimon** [comment]
@juliushaertl Sure I will try fix the docs, I did not noticed that this documents are generated.

**Comment by Cacodaimon** [comment]
@juliushaertl Finally got the docs fixed 🥳

File: rollup.config.js [hunk 1]
```diff
@@ -1,53 +1,60 @@
...
+	external: [
+		'unified',
+		'remark-parse',
+		'remark-disable-tokenizers',
+		'remark-external-links',
+		'remark-external-links',
+		'remark-rehype',
+		'rehype-add-classes',
+		'rehype-react',
+		'rehype-sanitize'
```
Comment (juliushaertl) [inline rollup.config.js:28]: Correct me if I'm wrong but it seems that rehype-sanitize is not used. We probably should either make use of that or add `'html'` as a default to the disableMarkdownBlockFeatures list

**Review by juliushaertl — CHANGES_REQUESTED** [review]
See inline, otherwise this seems good to me :+1: 

Great work so far :heart:

**Comment by juliushaertl** [comment]
> Cacodaimon requested a review from juliushaertl 2 minutes ago

What a timing :grin:

File: rollup.config.js (hunk 1 above)
Comment (Cacodaimon) [reply rollup.config.js]: I cleaned up the dependencies https://github.com/juliushaertl/vue-richtext/pull/226/commits/638a8bd1633530e25089704030e7b533c10a5b36

HTML is disabled by default: https://github.com/juliushaertl/vue-richtext/pull/226/commits/0fe95bb4fd1935e5f59225a05efe293bdcdf8822

**Comment by Cacodaimon** [comment]
Hey whats the state of this PR, is there anything I shall do?

**Comment by juliushaertl** [comment]
I've been on vacation the last two weeks, will have another look soonish :wink:

**Review by juliushaertl — APPROVED** [review]
Very nice contribution. Code looks good and works like a charm. Also the performance is still good from my POV. :+1:

**Comment by juliushaertl** [comment]
Release as 1.0.0 pending at https://github.com/juliushaertl/vue-richtext/actions/runs/348328237

**Comment by Cacodaimon** [comment]
> Very nice contribution. Code looks good and works like a charm. Also the performance is still good from my POV. +1

Thanks 😊, I will continue on my spreed/talk pull request for the MD support after I fixed my development environment.

**Comment by tromlet** [comment]
Forgive the noob developer question here but... what's the status of this PR on getting integrated into Nextcloud proper? Is it already part of NC20 or 21?

**Comment by nickvergessen** [comment]
This is a library that is used by some apps which are available for varius versions of Nextcloud.
So the main question is which place you are thinking about?

**Pull request merged**